test/e2e/specs/realrp/vnetpeering.go

```diff
@@ -0,0 +1,129 @@
+package realrp
+
+import (
+	"context"
+	"errors"
+	"os"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2018-06-01/network"
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2018-05-01/resources"
+	"github.com/sirupsen/logrus"
+
+	"github.com/openshift/openshift-azure/pkg/api/2018-09-30-preview/api"
+	"github.com/openshift/openshift-azure/pkg/fakerp/client"
+	"github.com/openshift/openshift-azure/test/clients/azure"
+)
+
+var _ = Describe("Peer Vnet tests [Vnet]", func() {
```

Comment:
Also [Real] to denote that it is meant to run using the production RP.

Reply:
Done.